org.openhab.binding.zigbee/README.md

```diff
@@ -8,19 +8,19 @@ The ZigBee binding supports an interface to a wireless ZigBee home automation ne
 
 A ZigBee Coordinator is the network controller, and is therefore the heart of the ZigBee network. It also acts as the trust centre to control security access to the network.
 
-Coordinators need to be installed manually and the serial port must be set.
+Coordinators need to be installed manually and the serial port and baud rate must be set. These are set to match the configuration of the dongle. Should you wish to use a different baud rate than the default, you must change the configuration of the dongle, and then reconfigure the binding.
```

Comment:
It is unclear to me what is meant with 'reconfigure the binding'. Shouldn't the change of the baud rate become effective when changing the configuration of the bridge?

Reply:
I added this statement because someone recently decided they wanted to change the Telegesis baud rate the 57k - so they changed the configuration in the binding, and then it stopped working. What I want to get across here is that the binding doesn't try and search for the baud rate and set it to a new value - this is a setting that must be compatible with the dongle and if you want to change the dongle speed (in instances where it is possible) then the dongle must first be reconfigured, and then the binding adjusted accordingly.